Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertPolygonElement.cs

```diff
@@ -11,6 +11,34 @@ namespace Objects.Converter.Revit;
 
 public partial class ConverterRevit
 {
+  public ApplicationObject GisFeatureToNative(GisFeature gisFeature)
```

Comment:
This is creating a tight coupling between Gis and Revit, are we happy for this? We did discuss that we need to be careful when picking support for these workflows. I guess we've done it for other things but might be worth discussing with @clairekuang as well.